File: Core/OBJECT/DatiAcquisiti.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Core.OBJECT
{
    public class DatiAcquisiti
    {
        public DatiAcquisiti(Sensore aSensore)
        {
            SENSORE = aSensore;
            DATA_ATTIVAZIONE = DateTime.Now;
        }
        public Sensore SENSORE { get; private set; }
        public DateTime DATA_ATTIVAZIONE { get; private set; }
        public DateTime DATA_DISATTIVAZIONE { get; set; }
        public double DifferenceSeconds()
        {
            return (DATA_DISATTIVAZIONE - DATA_ATTIVAZIONE).TotalSeconds;
        }
    }
}

File: ReactionTrainingPY/it/napalm/core/object/DatiAcquisiti.py
'''
Created on 18/mar/2015

@author: Luigi
'''
from datetime import datetime
class DatiAcquisiti(object):
    SENSORE = ''
    DATA_ATTIVAZIONE = 0
    DATA_DISATTIVAZIONE = 0

    def __init__(self, aSensore):
        self.SENSORE = aSensore;
        self.DATA_ATTIVAZIONE = datetime.now()
    
    def DifferenceSeconds(self):
        return (self.DATA_ATTIVAZIONE - self.DATA_ATTIVAZIONE).microseconds
        